vagrant/aws/aws-init.sh

```diff
@@ -18,16 +18,31 @@
 # or bring up your mini Kafka cluster.
 
 # Install dependencies
-sudo apt-get install -y maven openjdk-6-jdk build-essential \
-            ruby-dev zlib1g-dev realpath python-setuptools
+sudo apt-get install -y \
+  vagrant \
+  maven \
+  openjdk-8-jdk \
```

Comment:
Ubuntu 14.04 (the last LTS release which is still in its support phase) had jdk6 and jdk7, but no jdk8. Ubuntu 16.04 (the current LTS release, in support for 4 more years) apparently has jdk8 and jdk9, which is presumably why you needed to change the version. Unfortunately no overlap.

Anyway, my question here was more regarding what we consider the baseline for testing. In general, we've written all our scripts to build/test against the earliest version of dependencies that we support by default (e.g. both jdk and scala). I was just asking whether we wanted to be consistent with the jdk7 default until the switch is made for KIP-118 or if we're fine making this change preemptively.

So I guess the end result of this is that it might make sense to indicate minimum Ubuntu version in the instructions as well since people still use versions where jdk8 wasn't available.